external/storm-kafka/src/jvm/storm/kafka/SpoutConfig.java

```diff
@@ -17,6 +17,8 @@
  */
 package storm.kafka;
 
+import backtype.storm.utils.Utils;
```

Comment:
One of the committers will do that. But for that to happen, the PR should be ready for merge. Currently it has conflicts with master branch.